tests/NovelRT.Tests/Interop/GeoVector3FTest.cpp

```diff
@@ -173,5 +173,5 @@ TEST(C_GeoVector3F_Test, rotateToAngleAroundPointRotatesCorrectAmount) {
 }
 
 TEST(C_GeoVector3F_Test, geoVector2ConstructorReturnsCorrectGeoVector3) {
```

Comment:
Need to fix naming of Interop tests to be consistent (similar to `GeoVector4F`'s tests)

Reply:
Have I addressed this unintentionally?